relations/routes/api.py

```diff
@@ -21,3 +21,43 @@ def service_status() -> Response:
     response.status_code = status_code
     response.headers.extend(headers)
     return response
+
+@blueprint.route('/{str:arxiv_id_str}v{int:arxiv_ver}/relations',methods=['POST'])
+def create_new(arxiv_id_str: str,arxiv_ver: int) -> Response:
+    response_data,status_code,headers = controllers.create_new(arxiv_id_str,arxiv_ver,request.params)
```

Comment:
It doesn't match the return values of `controllers.create_new`. Please check below. Plus, the function may raise an `InternalServerError` exception.
https://github.com/arXiv/arxiv-external-links/blob/e29c191b0c68be9ac72cfbe570936dcfbe57e1bd/relations/controllers.py#L49-L58

Moreover, I am not sure whether you can return all of the 'Relation' values to users.